registry.k8s.io/images/k8s-staging-ingress-nginx/images.yaml

```diff
@@ -192,6 +192,7 @@
     "sha256:2d471b3a34dc43d10c3f3d7f2a6e8a2ecf7654a4197e56374261c1c708b16365": ["v0.0.12"]
 - name: nginx
   dmap:
+    "sha256:87dcd5ed0206161b86f3767ccdc592716617481e3eac89bfc46ec515419c0b94": ["v0.1.0"]
     "sha256:11ee0d0e3d063f1468f9a82958d57fa0718614fe10b676941f4dea0aef091faf": ["v1.0.0"]
```

Comment:
why we have a v1.0.0 and now downgrade to 0.1.0?

Reply:
We are not downgrading the `main`. This image was build on and is meant for the `release-1.xx` branches since they still include the Global Rate Limiting feature which already got removed from `main`.

Reply:
See marctl create backup Backup on Slack. 🙂

Reply:
ok thanks!